packages/sdk/src/types/Vouchers.ts

```diff
@@ -1,6 +1,24 @@
 import { OrdersGetResponse } from './Orders'
 import { SimpleCustomer } from './Customers'
 
+export interface SimpleVoucher {
+	code_config?: {
+		length?: number
+		charset?: string
+		pattern?: string
+	}
+	type?: string
```

Comment:
This should not be `?` and we have a good list of possible values:
`type VoucherType = 'DISCOUNT_VOUCHER' | 'GIFT_VOUCHER' | 'LOYALTY_CARD' | 'LUCKY_DRAW_CODE'` -> this is in `Distributions.ts`, please move it here to `Vouchers.ts` as it maks more sense and just import in in `Distributions.ts`